tests/storage/test_storage.py

```diff
@@ -175,6 +175,28 @@ def test_list_files(self, get_boto_bucket):
             Prefix="folder/a/b"
         )
 
+    @mock.patch.object(storage.S3FileStore, "_get_boto_bucket")
+    def test_list_files_no_path(self, get_boto_bucket):
+        get_boto_bucket.return_value = mock.Mock(
+            **{
+                "objects.filter.return_value": [
+                    mock.Mock(key="folder/a/b/"),
+                    mock.Mock(key="folder/a/b/foo.txt"),
+                    mock.Mock(key="folder/a/b/bar.txt"),
+                ]
+            }
+        )
+        store = storage.S3SubdirectoryFileStore("s3://some-bucket")
+
+        assert list(store.list_files(namespace="folder/a/b")) == [
```

Comment:
Just to confirm - I can safely assume that I don't need a trailing for prefixing `/` for the namespace, right?

Reply:
Yea you shouldn't need a trailing `/` for the namespace, that was a wrong assumption I made initially